src/app/components/Transactions/index.tsx

```diff
@@ -63,8 +63,8 @@ export const Transactions: FC<TransactionProps> = ({
     { content: t('common.type') },
     { content: t('common.from'), width: '150px' },
     { content: t('common.to'), width: '150px' },
-    { content: t('common.txnFee'), align: TableCellAlign.Right },
-    { align: TableCellAlign.Right, content: t('common.value') },
+    { content: t('common.txnFee'), align: TableCellAlign.Right, width: '250px' },
```

Comment:
Why is it that some of the columns headers are aligned to the right, but not all?

Reply:
Yes, I understand the point of right-aligning the numbers. My question is about the column headers. Especially there: 
![image](https://user-images.githubusercontent.com/2093792/220629530-39939157-c5f7-4240-bcf6-72ed57472051.png)

It's a question of personal aesthetics, but I would probably have the column headers aligned uniformly to the middle.  Some them being left and some right feels random to me. Especially "Age" vs "Type", and then "From" -> "To".  Because of the arrow connecting that pair of fields, I would probably align the "from" header to the right, so that both the "from" and the "to" are close to the arrow.

But again, personal asthetics.